pkg/action/util/util.go

```diff
@@ -545,3 +546,36 @@ func FindPodsByController(kubeClient *client.Clientset, namespace, contName, con
 	}
 	return pods, nil
 }
+
+// check whether parentKind is supported for MovePod/ResizeContainer actions
+// currently, these actions can only works on barePod, ReplicaSet, and ReplicationController
+//   Note: pod's parent cannot be Deployment.
```

Comment:
Could you please add comments how it works if a pod is controlled by Deployment?